### PR TITLE
Integration of HP Admin UI

### PR DESCRIPTION
**Pull request by peeech**

### CODE REVIEW

**Comment by peeech** [comment]
cc: @yegortimoshenko

File: modules/services/hp-admin-crypto-server.nix [hunk 1]
```diff
@@ -20,6 +20,8 @@ in
     systemd.services.hp-admin-crypto-server = {
       wantedBy = [ "multi-user.target" ];
 
+      environment.RUST_LOG = "debug";
```
Comment (yegortimoshenko) [inline modules/services/hp-admin-crypto-server.nix]:
```suggestion
      environment.RUST_LOG = "info";
```

`debug` is intended only for development purposes.

File: overlays/holo-nixpkgs/hpos-admin/hpos-admin.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def cas_hash(data):
     return b64encode(sha512(dump.encode()).digest()).decode()
 
 
-@app.route('/v1/config', methods=['GET'])
+@app.route('/api/v1/config', methods=['GET'])
```
Comment (yegortimoshenko) [inline overlays/holo-nixpkgs/hpos-admin/hpos-admin.py]: `/api` is a routing-specific implementation detail. Could you make Nginx mount `hpos-admin` with `/api` prefix? https://stackoverflow.com/a/32550251

File: profiles/logical/hpos/default.nix [hunk 3]
```diff
@@ -107,31 +107,62 @@ in
...
+              return 404;
+            '';
+        };
+
+        "~ ^/holofuel(?:/.*)?$" = {
+            extraConfig = ''
+              rewrite ^/holofuel.*$ / last;
+              return 404;
+            '';
+        };
```
Comment (yegortimoshenko) [inline profiles/logical/hpos/default.nix]: This is really flaky. I'd very much prefer hash-based routing in lieu of this, but I guess we don't have much time to argue about merits of that.

File: profiles/logical/hpos/default.nix [hunk 4]
```diff
@@ -107,31 +107,62 @@ in
...
+      extraConfig = ''
+        types {
+          application/javascript js;
+          application/wasm wasm;
+          image/png png;
+          image/svg+xml svg;
+          text/css css;
+          text/html html;
+        }
+      '';
```
Comment (yegortimoshenko) [inline profiles/logical/hpos/default.nix]: Again, good enough for this PR, but really, we should patch Nginx derivation to include `application/wasm wasm` MIME type.

File: modules/services/hp-admin-crypto-server.nix (hunk 1 above)
Comment (peeech) [reply modules/services/hp-admin-crypto-server.nix]: Related idea described in #371

File: profiles/logical/hpos/default.nix (hunk 3 above)
Comment (peeech) [reply profiles/logical/hpos/default.nix]: Don't think that we have time for this kind of redesign. At least flakiness is contained to 2 regex expressions.

File: profiles/logical/hpos/default.nix (hunk 4 above)
Comment (peeech) [reply profiles/logical/hpos/default.nix]: I'm gonna give it a quick try, will leave as is if not successfull

File: overlays/holo-nixpkgs/hpos-admin/hpos-admin.py (hunk 2 above)
Comment (peeech) [reply overlays/holo-nixpkgs/hpos-admin/hpos-admin.py]: Actually I would go even further - mount entire hpos-admin.py at `/api/v1/` as a v1 version of api, then inside API server itself have routes like: `/config`, `/settings` etc. This allows for multiple versions running simultaneously on one server!

Comment (yegortimoshenko) [reply overlays/holo-nixpkgs/hpos-admin/hpos-admin.py]: :+1:

File: profiles/logical/hpos/default.nix (hunk 4 above)
Comment (peeech) [reply profiles/logical/hpos/default.nix]: Done in [c680621](https://github.com/Holo-Host/holo-nixpkgs/pull/365/commits/c680621e06a3164fb87e12848d93e67888c0ed09)

File: overlays/holo-nixpkgs/hpos-admin/hpos-admin.py (hunk 2 above)
Comment (peeech) [reply overlays/holo-nixpkgs/hpos-admin/hpos-admin.py]: Done in [c680621](https://github.com/Holo-Host/holo-nixpkgs/pull/365/commits/c680621e06a3164fb87e12848d93e67888c0ed09)

File: profiles/logical/hpos/default.nix (hunk 4 above)
Comment (peeech) [reply profiles/logical/hpos/default.nix]: Resolved in #374

**Pull request merged**